Find a functionally equivalent code:com.android.keyguard.KeyguardUpdateMonitor.handleKeyguardVisibilityChanged:COMMENT
Method Modifier: private     
Comment:/**
 * Handle {@link #MSG_KEYGUARD_VISIBILITY_CHANGED}
 */

@@ -6,12 +6,12 @@
     for (int i = 0; i < mCallbacks.size(); i++) {
         KeyguardUpdateMonitorCallback cb = mCallbacks.get(i).get();
         if (cb != null) {
-            cb.onKeyguardVisibilityChanged(isShowing);
+            cb.onKeyguardVisibilityChangedRaw(isShowing);
         }
     }
 }

